hoot-core-test/src/test/cpp/hoot/core/io/OsmGeoJsonWriterTest.cpp

```diff
@@ -78,21 +82,23 @@ class OsmGeoJsonWriterTest : public HootTestFixture
     Log::WarningLevel logLevel = Log::getInstance().getLevel();
     if (Log::getInstance().getLevel() >= Log::Info)
       Log::getInstance().setLevel(Log::Error);
-    runTest("test-files/BostonSubsetRoadBuilding_FromOsm.osm", "BostonSubsetRoadBuilding.geojson");
+    Settings s;
+    s.set(ConfigOptions::getJsonFormatHootenannyKey(), false);
+    runTest("test-files/BostonSubsetRoadBuilding_FromOsm.osm", "BostonSubsetRoadBuilding.geojson", &s);
     Log::getInstance().setLevel(logLevel);
   }
 
   void runObjectGeoJsonTest()
   {
-    runTest(_inputPath + "SampleObjectsWriter.osm", "SampleObjectsWriter.geojson");
+    Settings s;
+    s.set(ConfigOptions::getJsonFormatHootenannyKey(), false);
+    runTest(_inputPath + "SampleObjectsWriter.osm", "SampleObjectsWriter.geojson", &s);
   }
 
   void runObjectGeoJsonHootTest()
   {
     Settings s;
     s.set(ConfigOptions::getJsonPrettyPrintKey(), true);
-    s.set(ConfigOptions::getJsonFormatHootenannyKey(), true);
-
```

Comment:
Since Hootenanny format is the default this isn't necessary.  It used to be `true` until the settings changed it to `false`.